Source/FieldSolver/FiniteDifferenceSolver/FiniteDifferenceAlgorithms/CartesianYeeAlgorithm.H

```diff
@@ -147,6 +166,30 @@ struct CartesianYeeAlgorithm {
 #endif
     }
 
+    /**
+     * Perform derivative along y on a nodal grid, from a cell-centered field `F`*/
+    template< typename T_Field>
+    AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE
+    static amrex::Real Dyy (
+        T_Field const& F,
+        amrex::Real const * const coefs_y, int const n_coefs_y,
```

Comment:
For consistency with `Dxx`:
```suggestion
        amrex::Real const * const coefs_y, int const /*n_coefs_y*/,
```